libs/wire-api/src/Wire/API/Provider.hs

```diff
@@ -56,7 +56,7 @@ import Data.Id
 import Data.Misc (HttpsUrl (..), PlainTextPassword6, PlainTextPassword8)
 import Data.Range
 import Data.Schema
-import Data.Swagger qualified as S
+import Data.OpenApi qualified as S
```

Comment:
was that an oversight?  shouldn't Data.Swagger be unavailable at this point?  (probably a topic for a separate PR.)

Reply:
We've moved away from it to openapi3 a while back.